Random NPE crash on CompositionPopupWindow

**Describe the bug**
A clear and concise description of what the bug is.
Random crash on CompositionPopupWindow.

**To Reproduce**
Random crash when open App.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Log**
If applicable, add crash log to quick focus your problem.
--------- Crash stacktrace
java.lang.RuntimeException: Unable to stop service com.osfans.trime.TrimeImeService@6048119: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.osfans.trime.ime.text.CompositionPopupWindow.destroy()' on a null object reference
	at android.app.ActivityThread.handleStopService(ActivityThread.java:4706)
	at android.app.ActivityThread.-$$Nest$mhandleStopService(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2185)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7924)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.osfans.trime.ime.text.CompositionPopupWindow.destroy()' on a null object reference
	at com.osfans.trime.ime.core.Trime.onDestroy(Trime.java:507)
	at android.app.ActivityThread.handleStopService(ActivityThread.java:4686)
	... 9 more

--------- beginning of crash
01-18 21:04:49.047 E/AndroidRuntime(15867): FATAL EXCEPTION: main
01-18 21:04:49.047 E/AndroidRuntime(15867): Process: com.osfans.trime, PID: 15867
01-18 21:04:49.047 E/AndroidRuntime(15867): java.lang.RuntimeException: Unable to stop service com.osfans.trime.TrimeImeService@6048119: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.osfans.trime.ime.text.CompositionPopupWindow.destroy()' on a null object reference
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at android.app.ActivityThread.handleStopService(ActivityThread.java:4706)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at android.app.ActivityThread.-$$Nest$mhandleStopService(Unknown Source:0)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2185)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at android.os.Handler.dispatchMessage(Handler.java:106)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at android.os.Looper.loopOnce(Looper.java:201)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at android.os.Looper.loop(Looper.java:288)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at android.app.ActivityThread.main(ActivityThread.java:7924)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at java.lang.reflect.Method.invoke(Native Method)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
01-18 21:04:49.047 E/AndroidRuntime(15867): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.osfans.trime.ime.text.CompositionPopupWindow.destroy()' on a null object reference
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at com.osfans.trime.ime.core.Trime.onDestroy(Trime.java:507)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	at android.app.ActivityThread.handleStopService(ActivityThread.java:4686)
01-18 21:04:49.047 E/AndroidRuntime(15867): 	... 9 more
01-18 21:04:49.076 I/om.osfans.trime(15867): System.exit called, status: 10
01-18 21:04:49.076 I/AndroidRuntime(15867): VM exiting with result code 10, cleanup skipped.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Smartphone (please complete the following information):**
 - Device: [e.g. Xiaomi Mix 4]
 - OS: [e.g. Android 11]
 - Theme: [e.g. Tongwenfeng]
 - Version: [e.g. 3.2.1]
 - Build info: 
   ```
   You could find build info in About Activity and click it to copy.
   e.g. 
   Build: Commit CI
   Date: 2022-06-12 10:09 UTC
   Commit: v3.2.6-58-gd9b4c1f, 2022-06-12 17:58:14 +0800
   Branch: develop
   Repository: https://github.com/osfans/trime
   ```
**Additional context**
Add any other context about the problem here.

--- a/app/src/main/java/com/osfans/trime/ime/core/Trime.java
+++ b/app/src/main/java/com/osfans/trime/ime/core/Trime.java
@@ -504,7 +504,9 @@ public class Trime extends LifecycleInputMethodService {
       listener.onDestroy();
     }
     eventListeners.clear();
-    mCompositionPopupWindow.destroy();
+    if (mCompositionPopupWindow != null) {
+      mCompositionPopupWindow.destroy();
+    }
     super.onDestroy();
 
     self = null;

